### PR TITLE
basic breadcrumb component

### PR DESCRIPTION
**Pull request by johnang-del**
basic breadcrumb component

### CODE REVIEW

File: src/components/Breadcrumb/Breadcrumb.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	render() {
+		const { history } = this.props;
+
+		return (
+			<div className="breadcrumb">
+				{history.map((item, index) => {
+					return (
+						<React.Fragment key={index}>
+							{item.isActive ? (
+								<div className="breadcrumb--object">{item.title}</div>
```
Comment (pauvalenzuela) [inline src/components/Breadcrumb/Breadcrumb.js]: This is the element inside the block of the 'breadcrumb' right? so it should be breadcrum__object

we '--' used this for the modifiers like the sizes, colors, actions.

File: src/components/Breadcrumb/Breadcrumb.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+							) : (
+								<a href={item.path} className="breadcrumb--object breadcrumb--link">
```
Comment (pauvalenzuela) [inline src/components/Breadcrumb/Breadcrumb.js]: This should be breadcrumb__link. and if you're using same style in the breadcrum__object for the link and the container it should be extend in the css not used same classNames.

Comment (johnang-del) [reply src/components/Breadcrumb/Breadcrumb.js]: the styling is different for __link and __object in terms of font weight to highlight difference between previous path to current path

**Comment by johnang-del** [comment]
Screen Shot 2019-08-13 at 12 53 22 PM

**Pull request merged**